xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -56,7 +56,7 @@ To be able to format the output with indentation and white space OR to skip vali
       <Docs>
         <param name="bufferWriter">The destination for writing JSON text.</param>
         <param name="options">Defines the customized behavior of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> By default, it writes minimized JSON (with no extra whitespace) and validates that the JSON being written is structurally valid according to the JSON RFC.</param>
-        <summary>Constructs a new <see cref="T:System.Text.Json.Utf8JsonWriter" /> instance with a specified <see langword="System.Buffers.IBufferWriter&lt;System.Byte&gt;" />.</summary>
+        <summary>Constructs a new <see cref="T:System.Text.Json.Utf8JsonWriter" /> instance with a specified <see cref="T:System.Buffers.IBufferWriter`1" /> to write the output to.</summary>
```

Comment:
Somehow GitHub won't let me do a suggestion for this line but it should be something like:

`Initializes a new instance of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> class using the specified buffer and customization options.`

Reply:
It happens to me to sometimes, @mairaw. Press Control+G on the comment textbox, or manually add the suggestion text. Here's your suggestion:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> class using the specified buffer and customization options.</summary>
```

Reply:
IBufferWriter isn't a buffer though. I would prefer to keep that as is, and also specify that the IBW is where we are "writing the output to".

Any better way to word it to keep both those things?